parts/kubernetesmasteraddons-tiller-deployment1.5.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
```

Comment:
The API differences between k8s v1.5.x and later versions are substantial enough to warrant a new deployment spec. Hopefully soon we'll be able to remove k8s v1.5.x entirely from acs-engine.